### PR TITLE
Add Travis CI, Appveyor and Debian packaging files

### PR DESCRIPTION
**Pull request by garbear**
Copied from peripheral.joystick.

### CODE REVIEW

**Comment by garbear** [comment]
@wsnipex is there a way to install the udev rule in this repo with debian packaging?

**Comment by wsnipex** [comment]
see https://github.com/kodi-game/peripheral.xarcade/pull/5 for the udev rule and some cleanup

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+version: BuildNr.{build}
```
[the remark this answers is not in the document]
Comment (garbear) [reply appveyor.yml]: Does that mean delete this file? I don't have any experience with appveyor, so if it needs changes can you PR?

[the remark this answers is not in the document]
Comment (Rechi) [reply appveyor.yml]: appveyor.yml is for windows ci

**Comment by garbear** [comment]
@Rechi updated .travis.yml. Can you enable travis? I can't get it to see my `packaging` branch.

**Comment by Rechi** [comment]
How should I do it? I'm no an admin (not even a member) of this organisation.

**Comment by garbear** [comment]
Want an invite? We could use another pair of eyes on our plethora of add-ons.

**Pull request merged**